pkg/util/util.go

```diff
@@ -93,16 +93,24 @@ func StartSPDKTgtDaemon() error {
 	return nil
 }
 
+func ForceStopSPDKTgtDaemon(timeout time.Duration) error {
+	return stopSPDKTgtDaemon(timeout, syscall.SIGKILL)
+}
+
 func StopSPDKTgtDaemon(timeout time.Duration) error {
+	return stopSPDKTgtDaemon(timeout, syscall.SIGTERM)
+}
+
+func stopSPDKTgtDaemon(timeout time.Duration, signal syscall.Signal) error {
```

Comment:
NIT: Can we make this a general process stop function and put it in [longhorn/go-common-libs](https://github.com/longhorn/go-common-libs)?

Reply:
Sure. Can we handle it later? While working on the fix, I found some functions can move to the go-common-libs. I will create a refactor task later.

Reply:
Sure!